pygmt/src/grdfill.py

```diff
@@ -125,14 +125,17 @@ def grdfill(
     r"""
     Interpolate across holes in a grid.
 
+    .. note::
+
+       Wraps the GMT module ``grdfill``.
+       The GMT documentation is at :gmt-docs:`grdfill.html`.
+
```

Comment:
Hm. If it should fit into one line, maye we can put the link to the docs within the ``grdfill``.  
Not sure, if the link works in this way or the full URL is required. 

```suggestion
    .. note::

       Wraps the GMT module [``grdfill``](:gmt-docs:`grdfill.html`).

```